index.bs

```diff
@@ -2643,19 +2664,119 @@ To <dfn>check if an [=attribution source=] exceeds the time-based destination li
+    1. [=Assert=]: |record|'s [=attribution rate-limit record/destination limit priority=] is not null.
+    1. [=Assert=]: |record|'s [=attribution rate-limit record/entity ID=] is not null.
+    1. Let |destinationRecord| be a new [=destination limit record=] struct whose items are:
+
+        : [=destination limit record/attribution destination=]
+        :: |record|'s [=attribution rate-limit record/attribution destination=]
+        : [=destination limit record/priority=]
+        :: |record|'s [=attribution rate-limit record/destination limit priority=]
+        : [=destination limit record/time=]
+        :: |record|'s [=attribution rate-limit record/time=]
+        : [=destination limit record/source identifier=]
+        :: |record|'s [=attribution rate-limit record/entity ID=]
+
+    1. [=list/Append=] |destinationRecord| to |destinationRecords|.
+1. [=set/iterate|For each=] [=site=] |destination| of |source|'s [=attribution source/attribution destinations=]:
+    1. Let |destinationRecord| be a new [=destination limit record=] struct whose items are:
+
+        : [=destination limit record/attribution destination=]
+        :: |destination|
+        : [=destination limit record/priority=]
+        :: |source|'s [=attribution source/destination limit priority=]
+        : [=destination limit record/time=]
+        :: |source|'s [=attribution source/source time=]
+        : [=destination limit record/source identifier=]
+        :: |record|'s [=attribution source/source identifier=]
+
+    1. [=list/Append=] |destinationRecord| to |destinationRecords|.
+1. [=list/sort in descending order|Sort=] |destinationRecords| in descending order, with |a| less than |b| if the following steps return true:
+    1. If |a|'s [=destination limit record/priority=] is less than |b|'s [=destination limit record/priority=], return true.
+    1. If |a|'s [=destination limit record/priority=] is greater than |b|'s [=destination limit record/priority=], return false.
+    1. If |a|'s [=destination limit record/time=] is less than |b|'s [=destination limit record/time=], return true.
+    1. If |a|'s [=destination limit record/time=] is greater than |b|'s [=destination limit record/time=], return false.
+    1. If |a|'s <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a>
+        [=destination limit record/attribution destination=]
+        is less than |b|'s <a href="https://html.spec.whatwg.org/multipage/origin.html#serialization-of-a-site">serialized</a>
+        [=destination limit record/attribution destination=], return true.
+    1. Return false.
+1. Let |sourcesToDelete| be a new [=set=].
+1. [=set/iterate|For each=] [=destination limit record=] |record| of |destinationRecords|:
+    1. Let |destination| be |record|'s [=destination limit record/attribution destination=].
+    1. If |newDestinations|'s [=set/size=] is less than the user agent's [=max destinations covered by unexpired sources=],
```

Comment:
Where is `newDestinations` declared?

Reply:
Hmm this may be accidentally removed in a merge, thanks for catching.